Automattic/pocket-casts-android
# Transcripts - Display in new UI

## Description

This 
- Removes transcript tab
- Displays the transcript in the new UI
- Adds gradient at the top and bottom to the new UI
- Adds transcript view to the back stack to allow dismissing it on back gesture (earlier, dragging down the transcript view would close the full-screen player. It caused issues in scrolling with a scrollable compose view inside the bottom sheet's nested scroll view.)

## Testing Instructions

1. Play an episode from "Cautionary Tales" (https://pcast.pocketcasts.net/8zpmjiru)
2. Open player tab
3. ✅ Notice there's no `Transcripts` tab
4. Tap on the `Transcript` bottom shelf item
5. ✅ Notice that the episode's transcript is displayed on the new layout
6. Scroll the layout
7. ✅ Notice that the transcript text scrolls behind the close button
8. Perform a back gesture or tap the close button
9. ✅ Notice that the transcript view is dismissed

Known issues:
- Player controls on tablet landscape are not available
- Investigate `RecyclerView does not support scrolling to an absolute position. Use scrollToPosition instead` in logs on animation
- Transcript auto-scrolling is not disabled while an episode is playing

## Screenshots or Screencast 


https://github.com/user-attachments/assets/7a1b37e8-8b37-4fb7-a059-c006bc6d19e6



## Checklist
 
#### I have tested any UI changes...
